src/actions/users.js

```diff
@@ -166,14 +167,16 @@ function completeLogin(data) {
             return {error};
         }
 
+        const serverVersion = Client4.getServerVersion();
+
         const promises = [
-            getMyPreferences()(dispatch, getState),
-            getMyTeams()(dispatch, getState),
+            dispatch(getMyPreferences()),
+            dispatch(getMyTeams()),
         ];
 
-        const state = getState();
-        const serverVersion = Client4.getServerVersion();
-        if (!isMinimumServerVersion(serverVersion, 4, 7) && getConfig(state).EnableCustomEmoji === 'true') {
+        if (isMinimumServerVersion(serverVersion, 5, 0)) { // should probably be 5.1 depending on when the version is updated on master
+            promises.push(dispatch(getClientConfig()));
+        } else if (!isMinimumServerVersion(serverVersion, 4, 7) && getConfig(getState()).EnableCustomEmoji === 'true') {
```

Comment:
Why is this an else block here?

Reply:
If the version is >= 5.0, then it will fail the second check anyway since that's looking for a version before 4.7. I could change it to make the code clearer though